client/lib/wpcom-undocumented/lib/site.js

```diff
@@ -86,18 +86,8 @@ function UndocumentedSite( id, wpcom ) {
 	this._id = id;
 }
 
-UndocumentedSite.prototype.domains = function( callback ) {
-	return this.wpcom.req.get( `/sites/${ this._id }/domains`, { apiVersion: '1.2' }, function(
-		error,
-		response
-	) {
-		if ( error ) {
-			callback( error );
-			return;
-		}
-
-		callback( null, response );
-	} );
+UndocumentedSite.prototype.domains = function() {
```

Comment:
You can always leave `callback` in here. If it's empty, it's going to return a Promise anyway, tho safer like this. :)

Reply:
Unless we missed a case. :laughing:

Reply:
TBH, we should use Promises, so let's not leave a "backdoor" for using callbacks.